person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
```

Comment:
- [ ] I think you should use the variable `@parent_permission`, instead of `parent_permission`.

Reply:
@DammyShittu,
Thanks for the review.
I just correct the mistake.